SDK/ViewControllers/Settings/SettingsViewController.swift

```diff
@@ -94,13 +94,13 @@ final class SettingsViewController: UIViewController {
     
     private let purchasesItems = [Item(title: "Restore Purchases")]
     
-    /// Contact menu items.
+    private static let supportEmail: String = Bundle.main.object(forInfoDictionaryKey: "CalendarStoreFeedbackEmailAddress") as? String ?? "support@schedjoules.com"
```

Comment:
There is no need to keep this in a static field.

Reply:
It's in a static property to avoid writing `Bundle.main.object(forInfoDictionaryKey: "CalendarStoreFeedbackEmailAddress") as? String ?? "support@schedjoules.com"` two times in lines 102 and 103

Reply:
Sure, but why has it to be static if none of the other members are static?

Reply:
Because otherwise we get this error:
![image](https://user-images.githubusercontent.com/2679333/65496299-d1951f00-de7d-11e9-9200-af725cdf945d.png)


The other option is to turn `contactItems` into a computed property and change it from `let` to `var`

```
private let supportEmail: String = Bundle.main.object(forInfoDictionaryKey: "CalendarStoreFeedbackEmailAddress") as? String ?? "support@schedjoules.com"
    private var contactItems: [Item] {
        get {
            [Item(title: "FAQ",
                  details: nil,
                  data: URL(string: "https://cms.schedjoules.com/static_pages/help_\(Locale.preferredLanguages[0].components(separatedBy: "-")[0]).html")),
             Item(title: "e-mail",
                  details: supportEmail,
                  data: supportEmail),
             Item(title: "Twitter",
                  details: "@schedjoules",
                  data: URL(string:"https://twitter.com/SchedJoules")),
             Item(title: "Facebook",
                  details: "SchedJoules",
                  data: URL(string:"https://www.facebook.com/SchedJoules/")),
             Item(title: "Website",
                  details: "http://www.schedjoules.com",
                  data: URL(string:"http://www.schedjoules.com"))]
        }
    }
```